qupulse/_program/waveforms.py

```diff
@@ -262,6 +327,7 @@ def __repr__(self):
 
 class ConstantWaveform(Waveform):
 
+    # TODO: remove
     _is_constant_waveform = True
```

Comment:
@peendebak can this be removed or would you rather keep it around?

Reply:
@terrorfisch Can be removed once I updated our stack. It is fine to deprecate it right away